doc/neps/nep-0013-ufunc-overrides.rst

```diff
@@ -173,12 +173,12 @@ where in all current cases only a single output makes sense).
 
 The function dispatch proceeds as follows:
 
-- If one of the input or output arguments implements
+- If one of the input, output or ``where`` arguments implements
```

Comment:
looks good but I think it is probably good to mention somewhere (on top?) that this adjustment has been made. Not quite sure what the proper protocol for that is! @seberg?

Reply:
Agreed, the fact that this NEP says "Status: Final" at the top says to me that this document is sort of supposed to be immutable right?

Reply:
Indeed, but I think this counts as a "clarification" -- in the end, we want people to be able to look at the NEP and be able to get how things work, and why things have changed. In astropy, we use a `data-last-revised` for that purpose -- see https://github.com/astropy/astropy-APEs#updating-apes

But numpy seems less clear. Two possibilities:
1. Add yourself as `author` and then insert the present date; I think this should be below "status: final" so that it is clear you made a small change.
2. Just add something like
```
.. note: This NEP was amended by <you> on <present date> to reflect the decision to allow 
       override also by `where` arguments.
```
I'd tend to go with (2) as being clearer...

Reply:
@mhvk, I looked through all the NEPs to see what we've done in the past and it seems like the closest thing was the `:Updated:` tag, so added that with myself as an author below the Status:Final line.